src/kapil/BankTrans.java

```diff
@@ -0,0 +1,94 @@
+/*Assignment 9 : 2nd April 2021
+
+Create a Banking System which has the following functionality.
+
+Create two object to perform below scenario.
+1) Method to debit amount
+2) Method to credit amount
+3) Method to printBalance [it will print current balance]
+4) individualTransactionSummary() method should show how many times individually debit, credit & printBalance method called .
+Hint : Create 3 non static instance variable.
+Output :
+user1 transaction summary : Credit - 2 times, Debit - 1 times, printBalance - 1 time
+user2 transaction summary : Credit - 5 times, Debit - 2 times, printBalance - 0 time
+
+5) allTransactionSummary() method should show by both users total how many times debit, credit & printBalance method get called.
+Hint : Create 3 static variables will be required.
+Output :
+All transaction summary : Credit - 7 times, Debit - 3 times, printBalance - 1 time
+*/
+
+package kapil;
+
+public class BankTrans {
+    String name;
+	int totalBal;
+	int debitCount = 0;
+	int creditCount = 0;
+    static int totalDebitCount = 0;
+    static int totalCreditCount = 0;
+    static int totalPrintBalance = 0;
+    
+    void setData(int initialBal, String cusName) {
+    	totalBal = initialBal;
+    	name= cusName;
+    }
+    
+	void debitAmt(int amount) {
+		if(amount <= totalBal) {
+			totalBal = totalBal - amount;
+			debitCount = debitCount + 1;	
+			totalDebitCount = totalDebitCount + 1;
+	}
+}
+
+	void creditAmt (int amount) {
+		if(amount <= totalBal ) {
```

Comment:
Why this require here?
while credits esa thodi hoga ki like If you have 2000 in ac and some want to x'fer 20000 in ac then he cannot x'fer to your acc.